lib/superstore.js

```diff
@@ -4,7 +4,7 @@
  * @author Matt Andrews <matthew.andrews@ft.com>
  * @copyright The Financial Times [All Rights Reserved]
  */
-
+require('es6-promise').polyfill();
```

Comment:
i don't generally ship promise implementations with deps, prefer BYO-Promise library approach.

e.g. https://github.com/matthew-andrews/denodeify
https://github.com/matthew-andrews/isomorphic-fetch

can put this line in the tests though ^^